### PR TITLE
Search

### PR DESCRIPTION
**Pull request by VyacheslavBukach**

### CODE REVIEW

File: lib/blocs/home_bloc/home_state.dart [hunk 1]
```diff
@@ -3,9 +3,19 @@ part of 'home_bloc.dart';
 @immutable
 abstract class HomeState {}
 
-class LoadingState extends HomeState {}
-
-class AuthenticatedState extends HomeState {}
+class InitialState extends HomeState {}
+
+class AuthenticatedState extends HomeState {
+  final Future<List<Book>> popularStream;
```
Comment (MaximSemashko) [inline lib/blocs/home_bloc/home_state.dart]: Incorrect name. Please remove word Stream. Can be names popularBooks. The same about releases

File: lib/presentation/screens/store_screen.dart [hunk 2]
```diff
@@ -118,3 +159,96 @@ class _FilterContainerState extends State<FilterContainer> {
     );
   }
 }
+
+class MySearchDelegate extends SearchDelegate {
```
Comment (MaximSemashko) [inline lib/presentation/screens/store_screen.dart]: please move to another file

File: lib/presentation/ui_components/horizontal_book_list.dart [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+import 'package:my_books/presentation/screens/book_detail_screen.dart';
 
-import '../../blocs/home_bloc/home_bloc.dart';
 import '../../domain/entities/book.dart';
 import 'horizontal_book_list_item.dart';
 
 class HorizontalBookList extends StatelessWidget {
   final double bookWidth;
-  final Future<List<Book>> bookList;
+  final Future<List<Book>> bookStream;
```
Comment (MaximSemashko) [inline lib/presentation/ui_components/horizontal_book_list.dart]: incorrect name

File: lib/blocs/home_bloc/home_state.dart (hunk 1 above)
Comment (VyacheslavBukach) [reply lib/blocs/home_bloc/home_state.dart]: done

File: lib/presentation/ui_components/horizontal_book_list.dart (hunk 3 above)
Comment (VyacheslavBukach) [reply lib/presentation/ui_components/horizontal_book_list.dart]: done

File: lib/presentation/screens/store_screen.dart (hunk 2 above)
Comment (VyacheslavBukach) [reply lib/presentation/screens/store_screen.dart]: done

**Pull request merged**